lib/tasks/db.rake

```diff
@@ -11,17 +11,17 @@ end
 
 module MultisiteTestHelpers
   def self.load_multisite?
-    Rails.env.test? && !ENV["RAILS_DB"] && !ENV["SKIP_MULTISITE"]
+    Rails.env.test? && !ENV["RAILS_TEST_DB"] && !ENV["SKIP_MULTISITE"]
   end
 
   def self.create_multisite?
-    (ENV["RAILS_ENV"] == "test" || !ENV["RAILS_ENV"]) && !ENV["RAILS_DB"] && !ENV["SKIP_MULTISITE"]
+    (Rails.env.test? || Rails.env.development?) && !ENV["RAILS_TEST_DB"] && !ENV["DATABASE_URL"] && !ENV["SKIP_MULTISITE"]
   end
 end
 
 task 'db:environment:set' => [:load_config]  do |_, args|
   if MultisiteTestHelpers.load_multisite?
-    system("RAILS_ENV=test RAILS_DB=discourse_test_multisite rake db:environment:set")
+    system("RAILS_ENV=test RAILS_TEST_DB=discourse_test_multisite rake db:environment:set")
```

Comment:
I'm not sure I understand this change here. `RAILS_DB` is an env that tells the rails_multisite gem which DB to connect to. By using a different environment variable, aren't we bypassing rails_multisite completely?

Reply:
> RAILS_DB is an env that tells the rails_multisite gem which DB to connect to. 

DB in the rails_multisite sense - which means a site. `discourse_test_multisite` is a postgres database. So yes, we do bypass gem's logic here - on purpose 🙂